ts/nni_manager/.eslintrc

```diff
@@ -24,14 +24,16 @@
         "@typescript-eslint/no-inferrable-types": 0,
         "no-inner-declarations": 0,
         "@typescript-eslint/explicit-function-return-type": "error",
+        "@typescript-eslint/no-var-requires": 0,
+        "@typescript-eslint/no-non-null-assertion": 0,
+
         "@typescript-eslint/no-unused-vars": [
-            "error",
+            "off",
```

Comment:
The latest version of eslint parser cannot handle typescript interfaces correctly and complains about unused vars.
Turn off this option until upstream fix.